src/api.tf

```diff
@@ -107,6 +107,12 @@ resource "azurerm_api_management_custom_domain" "api_custom_domain" {
   ]
 }
 
+resource "azurerm_api_management_notification_recipient_email" "email_assistenza_on_new_subscription" {
+  api_management_id = module.apim.id
+  notification_type = "RequestPublisherNotificationMessage"
+  email             = "cstar@assistenza.pagopa.it"
```

Comment:
Please put this value, into a var, so we can reference via variables files for each ENV